src/mappings/handlers/evmHandlers.ts

```diff
@@ -9,13 +11,314 @@ import { InvestorTransactionData, InvestorTransactionService } from '../services
+    shelf: [{ address: '0xE80C9e9fbaE9868e1D645f9727436afE5381047A' }],
+    pile: [{ address: '0xB7d1DE24c0243e6A3eC4De9fAB2B19AB46Fa941F' }],
+  },
+  {
+    id: '0xb5c08534d1e73582fbd79e7c45694cad6a5c5ab2',
+    shortName: 'BT2',
+    startBlock: 16047557,
+    navFeed: [{ address: '0xeff42b6d4527a6a2fb429082386b34f5d4050b2c', startBlock: 16047557 }],
+    reserve: [{ address: '0x86ade02617911d2bf020b29b3035e6b3d805992d', startBlock: 16047557 }],
+    shelf: [{ address: '0x77bd5e87fBdae67C495d17ba268DD9E494983627' }],
+    pile: [{ address: '0x611e36809ad4BB94ae6dE889fd4e830Fc21835f7' }],
+  },
+  {
+    id: '0xd8486c565098360a24f858088a6d29a380ddf7ec',
+    shortName: 'FNO1',
+    startBlock: 15460148,
+    navFeed: [{ address: '0x9ec3f3a40ba2572a48c8865e4aa89b3ee4ad4915', startBlock: 15460155 }],
+    reserve: [{ address: '0xc156aa23a29ef7678b4dec5b812b84797ed21ee0', startBlock: 15460175 }],
+    shelf: [{ address: '0x511680e8eD9cA463E5F1776281992de4Aec22039' }],
+    pile: [{ address: '0x8f7b701a7309c2Ef5c0Fe850ab7589d5b23a8f76' }],
+  },
+  {
+    id: '0xdb3bc9fb1893222d266762e9ff857eb74d75c7d6',
+    shortName: 'ConsolFreight 4',
+    startBlock: 11063000,
+    navFeed: [{ address: '0x69504da6b2cd8320b9a62f3aed410a298d3e7ac6', startBlock: 11063046 }],
+    reserve: [
+      { address: '0x0d601b451aFD502e473bA4CE6E3876D652BCbee7', startBlock: 11063150 },
+      { address: '0xfaec38ffee969cf18e88097ec62e30b70494e234', startBlock: 13004061 },
+    ],
+    shelf: [{ address: '0xA0B0d8394ADC79f5d1563a892abFc6186E519644' }],
+    pile: [{ address: '0x3fC72dA5545E2AB6202D81fbEb1C8273Be95068C' }],
+  },
+  {
+    id: '0xf96f18f2c70b57ec864cc0c8b828450b82ff63e3',
+    shortName: 'ALT 1.0',
+    startBlock: 14167745,
+    navFeed: [{ address: '0x6fb02533b264d103b84d8f13d11a4865ec96307a', startBlock: 14167760 }],
+    reserve: [{ address: '0x35f7de72c8597599ef4f2a18eafd162257301435', startBlock: 14167898 }],
+    shelf: [{ address: '0x11daC3fA9d2216377A79Bef04F6A2682630371c3' }],
+    pile: [{ address: '0xE18AAB16cC26EB23740D72875e0C6b52cEbb46b3' }],
+  },
+  {
+    id: '0xfc2950dd337ca8496c18dfc0256fb905a7e7e5c6',
+    shortName: 'databased.FINANCE 1',
+    startBlock: 11639815,
+    navFeed: [{ address: '0x00cd3ae59fdbd375a187bf8074db59edaf766c19', startBlock: 11639830 }],
+    reserve: [{ address: '0x729e12cdc0190a2e4ab4401bca4c16132d75adc5', startBlock: 11639877 }],
+    shelf: [{ address: '0xC42CfB07bC1140f9A615bD63c4fFAE5F8260Ab22' }],
+    pile: [{ address: '0xdB07B21109117208a0317adfbed484C87c9c2aFf' }],
+  },
+]
+
+export const handleEvmBlock = errorHandler(_handleEvmBlock)
+async function _handleEvmBlock(block: EthereumBlock): Promise<void> {
+  const date = new Date(Number(block.timestamp) * 1000)
+  const blockNumber = block.number
+  const newPeriod = (await timekeeper).processBlock(date)
+
+  const DAIAddress = '0x6b175474e89094c44da98b954eedeac495271d0f'
```

Comment:
This should be in `config.ts`, and should be named `DAIMainnetAddress` to be clear